src/i18n/locales/en.json

```diff
@@ -6,6 +6,23 @@
   "aysDescription": "Disaster Management System",
   "users": "Users",
   "home": "Home",
+  "page": "Page",
+  "admin": {
```

Comment:
Bu şekilde obje içinde obje olarak kullanım sonradan karışıklığa yol açıyor. admin objesinden çıkarabiliriz. Genel olarak statüler her yerde kullanılabilir. Tekrar tekrar tanımlamaya gerek olmaması ve kodların kısa olması için iç içe objelerden sakınalım.